compiler/onecc-docker/tests/onecc-docker_002.test

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Check if docker image is created
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+# run test
+onecc-docker -v
+RESULT=$(docker images -q onecc 2>&1)
+
+if [[ -z "$RESULT" ]]; then
+  trap_err_onexit
+fi
+
```

Comment:
```suggestion

# run test
onecc-docker -v
RESULT=$(docker images -q onecc:1.21.0 2>&1)

if [[ -z "$RESULT" ]]; then
  trap_err_onexit
fi

```

@jyoungyun 
How about this way?

Reply:
I'm sorry but I prefer the previous patch. Because this tc can be broken according to `one-compiler` version.

Reply:
I will proceed with the next work without modifying the 2nd tc Thank you!